### ISSUE TITLE
Keyboard navigation between data-grid header and rows should be done with the arrow keys not the tab

### ISSUE THREAD

**Issue by talimarcus** [vaadin/px-data-grid#468]
### Steps to reproduce
1. Tab to the data-grid header
2. Press the down arrow key

**Expected behavior:** Focus moves to the first item in the first row of the data-grid
**Actual behavior:** Nothing happens

### Discussion
The user shouldn’t have to tab between the column headers and the column data, that should be navigated using the arrow keys, as in the example here: https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html#

### Version
px-data-grid: v1.1.4
Browsers: Chrome 64, Firefox 57, Safari 11

**Comment by limonte** [comment]
The example you linked very simple and doesn't really matched the px-data-grid component which has always visible header. It's a good UX because users are always able to perform column actions like sorting, drag-n-drop, hiding, etc even if they scrolled the content.

Now, imagine the case: 
 - given a px-data-grid with 100 rows
 - 20 rows are in the viewport
 - user scrolled the content
 - "First name" header is focused 
![](https://i.imgur.com/3FSsE9l.png)
 - Now, the question: what should happen when user press Arrow Down? Should the first visible cell be focused (row # 9) or should the cell in the first row be selected?

In order to prevent the ambiguous behavior, Tab is used to navigate between the header and the content. With Tab user knows that the last active cell will be focused. If there's no last active cell, the very first cell will be focused.
